KinomeScan-Appyter/Kinase Input Data Processing

```diff
@@ -0,0 +1,69 @@
+# Creates the dictionaries and converts them to GMT files
+
+import pandas as pd
+
+old_url = 'http://lincs.hms.harvard.edu/db/datasets/20000/results?small+molecules=HMS_ID&output_type=.csv'
+
+df = []
+
+kinome_df = pd.read_excel('http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2013/11/HMS-LINCS_KinomeScan_Datasets_2018-01-18.xlsx')
+kinome_df['sm_hms_id'] = kinome_df['sm_hms_id'].str.replace(r'\D', '') #remove HMSL before the ID
+
+# retrieve the CSV file for the small molecules, 2-183
+for x in range(0, 181):  
```

Comment:
Generally speaking, `x` is a non-descriptive name.  It's ok in this case, but in bigger projects it tends to become a mess. Better version is something like `csv_name`.